README.md

```diff
@@ -45,6 +45,9 @@ Add your Bandwidth account settings to `.env`:
 * USERNAME
 * PASSWORD
 
+* WEBRTC_HTTP_SERVER_URL (Customer API URL, which defaults to `https://api.webrtc.bandwidth.com/v1`. Optionally override if you want to use your personal stack instead, i.e. `https://sife6x5c6l.execute-api.us-east-1.amazonaws.com/v1`)
+* WEBRTC_DEVICE_URL (Device Websocket API URL, which defaults to `wss://device.webrtc.bandwidth.com`. Optionally override if you want to use your personal stack instead, i.e. `wss://t7b04iwatb.execute-api.us-east-1.amazonaws.com`)
+
 Add your Voice Application settings to `.env`:
 * WEBRTC\_SIPX\_NUMBER (Bandwidth phone number used to transfer sip calls into WebRTC)
 * VOICE\_NUMBER (Bandwith phone number associated with the Voice Application in [E.164 format](https://www.bandwidth.com/glossary/e164/))
```

Comment:
The VOICE_API_ID is not used in the code. I debated removing it and the section above for creating a Voice Application but I left it because there is code for the voice callback URL and it looks like you need to create an application in order to register it